integration-examples/observability-microservices-jumpstart/README.md

```diff
@@ -86,7 +95,7 @@ Create kubernetes manifest files, making sure to export the RUM_REALM and TOKEN
 
 Run the command `kubectl get pods` and verify that the services are running and the external IP is exposed.
 
-## Wait for services to start and expost external ports
+## Wait for services to start and expose external ports
 
 After all services are running, use the following commands:
 
```

Comment:
Wondering if we can get a code block of what this would look like when everything is running and an example command? So users know what they are looking for in the `kubectl get pods` output.

Reply:
That sounds like a great idea, I will add some example output.

Reply:
👍